### PR TITLE
[Chore #160376364] Setup database config on existing endpoints

### PR DESCRIPTION
**Pull request by maxwellgithinji**
**What does this PR do?**
Setup databse on the existing endpoints with updated tests

**What are the relevant PT stories?**
https://www.pivotaltracker.com/story/show/160376364

### CODE REVIEW

File: API/database.ini [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[postgresql]
+host=localhost
+database=stackoverflow_lite_db
+user=root
+password=maxgit
```
Comment (Dhul-Husni) [inline API/database.ini]: integral data should be stored in the environment
i.e
user=os.env(blah blah)

File: API/helper/config_helper.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+def db_config(filename, section):
+    """
+    :param: filename: the name of the .ini file
+    :param: section:
+        - the section to look for
+          db config in the .ini file
+    :return:
+        - a database configuration object with the db configs
+    """
+    # create a parser
```
Comment (Dhul-Husni) [inline API/helper/config_helper.py]: comments not adding value :)

File: API/helper/config_helper.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    parser = ConfigParser()
+    # read config file
+    parser.read(filename)
```
Comment (Dhul-Husni) [inline API/helper/config_helper.py]: Here too

File: API/models/db_controler.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
+"""
+Database controler module to create or drop tables
```
Comment (Dhul-Husni) [inline API/models/db_controler.py]: nit: Database control`l`er

File: API/models/db_controler.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+class Db():
+    """
+    :class: DB: create or drop tables
+    """
+    def __init__(self, db_config):
+        """
+        :param db_config: database configuration object:
+        """
+        self.params = db_config
+        # connect to the PostgreSQL server
```
Comment (Dhul-Husni) [inline API/models/db_controler.py]: Here too

File: .gitignore [hunk 6]
```diff
@@ -102,3 +102,7 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+#db
+/API/manage.py
+/API/destroy.py
```
Comment (Dhul-Husni) [inline .gitignore]: why are we ignoring manage and destroy.py?

Comment (maxwellgithinji) [reply .gitignore]: I am using them locally to create and remove database while testing manually. but i will delete destroy.py and remove the manage.py from gitignore later on

**Comment by Dhul-Husni** [comment]
LGTM

**Pull request merged**